metadata-ingestion/src/datahub/ingestion/source/glue_profiling_config.py

```diff
@@ -0,0 +1,64 @@
+from pydantic.fields import Field
+
+from datahub.configuration.common import ConfigModel
+
+
+class GlueProfilingConfig(ConfigModel):
```

Comment:
Please, can you also add a table filter for profiling?
Similar to this:
https://github.com/datahub-project/datahub/blob/d0cdadbb3f4b89e771e2a05593ef528e622f94e5/metadata-ingestion/src/datahub/ingestion/source/sql/sql_common.py#L249

Reply:
Sure!